ci/helm-chart/Chart.yaml

```diff
@@ -15,9 +15,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.2.2
+version: 3.2.1
```

Comment:
Should this be 3.2.3?  Looks like it got downgraded.

Reply:
uhh fog brain, nice catch!